src/FileReader.java

```diff
@@ -0,0 +1,24 @@
+import java.io.File;
+import java.io.IOException;
+import java.io.RandomAccessFile;
+
+public class FileReader {
+
+    /*
+        Task reformat code of reading file.
+     */
+    public static void readFile(File file) {
+        String line = "";
+        try (RandomAccessFile input = new RandomAccessFile(file, "r")) {
```

Comment:
do not use try with resources))) feel how pretty code is without it)))

Reply:
Implemented without try with resources